src/main/java/org/hmhb/program/DefaultProgramService.java

```diff
@@ -307,6 +288,10 @@ public Program save(
             program.setOrganization(organizationService.getById(program.getOrganization().getId()));
         }
 
+        if (StringUtils.isBlank(program.getStreetAddress())) {
```

Comment:
When javadoc'ing, I noticed this exception wasn't being used.  It was a validation I forgot to write.